app/routes/dashboard.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+const router = express.Router();
+const recipesModel = require('../database/models/recipesModel');
+
+/* GET users listing. */
+router.get('/new_recipe', async function(req, res, next){
+    res.render('pages/newrecipe', {title: 'Create Recipe'});
+});
+
+router.post('/new_recipe', async function(req, res, next){
+    try {
+        const new_rec = req.body;
+        const uid = req.user.id;
+        await recipesModel.insertAsCreator(new_rec, uid);
+        return res.render('pages/AddRecipeSuccess', {title: 'Add Recipe Successful'});
+    } catch(err){
+        console.error(err);
+        return res.status(500)
+            .json({err, data: 'Unable to add recipe, internal server error'});  }
```

Comment:
Codacy has a fix for the issue: [Closing curly brace should be on the same line as opening curly brace or on the line after the previous block.](https://app.codacy.com/gh/cse110-fa21-group17/cse110-fa21-group17/pullRequest?prid=8445077)
```suggestion
            .json({err, data: 'Unable to add recipe, internal server error'});  
```